include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2949,9 +5112,28 @@ namespace internal
       constexpr unsigned int n_q_points_1d_actual =
         fe_degree > -1 ? n_q_points_1d : 0;
 
-      if (fe_degree > -1 &&
-          subface_index >= GeometryInfo<dim>::max_children_per_cell &&
-          shape_info.element_type <= MatrixFreeFunctions::tensor_symmetric)
+      if (fe_degree >= 1 &&
+          shape_info.element_type == MatrixFreeFunctions::tensor_raviart_thomas)
+        {
+          // TODO. Add symmetric_evaluation
```

Comment:
As said above, I would skip this for now, unless you feel it is very straight-forward to add (which I believe it is not).